src/Raven.Server/Documents/Handlers/CountersHandler.cs

```diff
@@ -199,6 +199,11 @@ void LoadDocument()
                 return CountersDetail.Counters.Count;
             }
 
+            public override TransactionOperationsMerger.IReplayableCommandDto<TransactionOperationsMerger.MergedTransactionCommand> ToDto(JsonOperationContext context)
+            {
+                throw new NotImplementedException();
```

Comment:
Why?

Reply:
Just for compiling

Reply:
We need to actually implement these, no?

Reply:
For compiling only

Reply:
done

Reply:
Okay, so that was the first pass, then you are going to impl these?

Reply:
done